EIPS/eip-3475.md

```diff
@@ -2,10 +2,9 @@
 eip: 3475
 title: Abstract Storage Bonds
 description: Interface for creating tokenized obligations with abstract on-chain metadata storage
-author: Yu Liu (@yuliu-debond), Varun Deshpande (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold (@Edoumou), Toufic Batrice (@toufic0710)
+author: Yu Liu (@yuliu-debond), Varun Deshpande (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold Edoumou (@Edoumou), Toufic Batrice (@toufic0710)
```

Comment:
@Edoumou I imagine this change is legit but I'd like to double-check this.

Reply:
sure so for that you need hisverification , @edoumou  can  verify you in DM .